lib/tasks/evm_ansible_runner.rake

```diff
@@ -0,0 +1,35 @@
+require 'awesome_spawn'
+require "vmdb/plugins"
+
+namespace :evm do
+  namespace :ansible_runner do
+    desc "Seed galaxy roles for provider playbooks"
+    task :seed do
+      plugins_with_req_yml = Vmdb::Plugins.select do |plugin|
+        req_yml_path = plugin.root.join('content_tmp', 'ansible', 'requirements.yml')
+        File.file?(req_yml_path)
+      end
+
+      plugins_with_req_yml.each do |plugin|
+        puts "Seeding roles for #{plugin.name}..."
+
+        roles_path = plugin.root.join('content_tmp', 'ansible', 'roles')
+        role_file  = plugin.root.join('content_tmp', 'ansible', 'requirements.yml')
```

Comment:
Would prefer `'content', 'ansible_tmp', ...`.  `content` won't change, but the inner dir might.

Reply:
Done